integration-tests/helpers/session.ts

```diff
@@ -1,5 +1,37 @@
 import {driver} from "@wdio/globals";
 
+const SHOW_SESSION_ID_TEXT_BUTTON = "~SHOW SESSION ID";
```

Comment:
the test harness will also need to be updated to make sure this button is shown on each screen and each of the existing test specs to use it before grabbing the span payloads

thinking more on this I do think it's simpler to instead DELETE the content in the namespace on the mock-server before each test. The downside of not being able to re-use the same app package concurrently in different tests runs I think is maybe less of a hassle in the long run, we'd need to make sure as part of CI/CD that we generate one APK per OS version we want to test and give them each a different namespace

Reply:
We can put that button on the app header or footer, so it will be there in any Page.